provider/vsphere/vm_template_test.go

```diff
@@ -13,14 +13,15 @@ import (
 	gc "gopkg.in/check.v1"
 
```

Comment:
Maybe I'm mistaken but I thought that any non juju/juju package should be in the same block as other 3rd party deps?